src/packagers/InMemoryPackager.py

```diff
@@ -0,0 +1,33 @@
+import os
+import io
+import requests
+from logging import Logger
+from zipfile import ZipFile, ZIP_LZMA
+from typing import List
+
+
+from src.providers import Provider
+from src.Product import Product
+from .Packager import Packager
+
+
+class InMemoryPackager(Packager):
+    def package(self, products):
+        with io.BytesIO() as mem_file:
+            zip_file = ZipFile(mem_file, mode='w', compression=ZIP_LZMA)
+            self._logger.info('Started Downloading')
+            for product in products:
+                product_descriptor = 'product {0}@{1} from provider {2}'.format(product.name, product.version, product.provider.name)
+                self._logger.debug('Downloading {0}'.format(product_descriptor))
+                path = '{0}/{1}/{2}.{3}'.format(product.provider.name,product.name, product.version, product.provider.file_ext)
```

Comment:
space after `product.provider.name,`... It really bothers me

Reply:
Fixed @PolaDinitz